templates/web/fixmystreet.com/footer_extra.html

```diff
@@ -31,7 +31,7 @@ <h2 class="mysoc-footer__site-name">FixMyStreet</h2>
                             %]>[% loc("Local alerts") %]</[% c.req.uri.path == '/alert' ? 'span' : 'a' %]></li>
                     </ul>
                     <ul>
-                        <li><a href="https://www.fixmystreet.com/council">FixMyStreet for Councils</a></li>
+                        <li><a href="/council">FixMyStreet for Councils</a></li>
```

Comment:
I wondered about that! If we change these to `/about/council`, should we also change all the other instances of `www.fixmystreet.org/council` in all the other cobrands?